src/style.css

```
@@ -0,0 +1,84 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Arial, sans-serif;
+}
+
+.container {
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  min-width: 300px;
+  color: white;
+  height: 300px;
+  justify-content: center;
+  align-items: center;
+  font-size: 100px;
+  flex-shrink: 0;
+  width: 33.33%;
```

```suggestion
  flex-basis: 33.33%;
```
Let's use flex-basis for this task